pages/index.tsx

```diff
@@ -24,57 +27,149 @@ export default function Home() {
-              ctaLink={section.ctaLink}
-              ctaText={section.ctaText}
-              ariaLabels={{ ctaButton: section.ariaLabels?.ctaButton }}
-            >
-              <Grid container spacing={4} id={`${section.id}-main-grid`}>
-                {section.items.map((item) => (
-                  <GridItemCard
-                    key={item.id}
-                    xs={12}
-                    md={section.items.length % 2 ? 4 : 3}
-                    title={item.title}
-                    body={item.body}
-                    imgsrc={item.imgsrc}
-                  />
-                ))}
+        <Grid
+          container
+          sx={{
+            backgroundColor: '#DCFAEF',
+          }}
+        >
+          <SectionContainer>
+            <Grid container justifyContent="space-between" alignItems="center">
+              <Grid item xs={12} md={8}>
+                <Typography
+                  variant="h3"
+                  sx={{ fontWeight: 500, textAlign: { xs: 'center', md: 'left' } }}
+                >
+                  {content.eligibilityCTA.title}
+                </Typography>
               </Grid>
-            </PaperSection>
+              <Grid item xs={12} md={4} sx={{ display: 'flex', justifyContent: { xs: 'center' }, mt: { xs: 2, md: 0 } }}>
+                <Button
+                  href="/calculator/head-initial-1-cont"
+                  variant="contained"
+                  color="primary"
+                  size="small"
+                  className="calc-btn"
+                  aria-label="Access our eligibility calculator"
+                  sx={{
+                    whiteSpace: 'nowrap',
+                    py: 1,
+                    px: 3,
+                    backgroundColor: '#003256',
+                  }}
+                >
+                  <Typography variant="button" sx={{ textTransform: 'none', marginBottom: '0px' }}>{content.eligibilityCTA.ctaButton}</Typography>
+                  <Box component="span" sx={{ ml: 1 }} />
+                  <ArrowForwardIcon sx={{ stroke: '#FFFEFC', strokeWidth: 1.5 }} />
+                </Button>
+              </Grid>
+            </Grid>
           </SectionContainer>
-        ))}
+        </Grid>
+
+        <SectionContainer>
+          <Grid container spacing={4}>
+            <Grid item xs={12} textAlign={{ xs: 'center', sm: 'left' }}>
+              <Typography variant="h2">How can we help you vacate a conviction?</Typography>
+            </Grid>
+            <Grid item xs={12} md={10}>
+              <ImageList sx={{
+                overflowX: 'auto', rowHeight: 'auto', display: 'flex', justifyContent: 'space-between', width: '100%',
+              }}
+              >
+                <ImageListItem sx={{
+                  display: 'flex', flexDirection: 'row', justifyContent: 'space-between', flexWrap: 'nowrap', width: '100%',
+                }}
+                >
+                  {content.carousel.map((card) => (
```

Comment:
The carousel looks a little off on larger screens, but I think adding the maxWidth of md or 724 to the whole page will help make things look better.

Reply:
@aitzeng